ext/oci8/php_oci8_int.h

```diff
@@ -531,6 +535,13 @@ ZEND_BEGIN_MODULE_GLOBALS(oci) /* {{{ Module globals */
 	char		*edition;
 ZEND_END_MODULE_GLOBALS(oci) /* }}} */
 
+/* {{{ transparent failover related prototypes */
+
+int php_oci_register_taf_callback(php_oci_connection *connection, zval *callback);
+int php_oci_disable_taf_callback(php_oci_connection *connection);
```

Comment:
I took a look at the other functions, do you mean to still return an int, but to return 1 on success and 0 on failure?